docs/wcs/examples/from_file.py

```diff
@@ -3,7 +3,7 @@
 
 from __future__ import division, print_function
```

Comment:
Should these also be removed (while you are looking at these files)?

Reply:
Good point. But it depends on the milestone for the PR. 

If we want to backport it, then I probably cannot remove it without breaking the 2.0.x branch.

Reply:
Good point. I'm not sure how could a few ``__future__ remained, but I suggest to leave them to a separate PR for 3.0.

Reply:
I think they avoided detection by being in a `.py` files in the `docs` directory...

Reply:
yes, probably that was the trick. Could you also check that no other python2 stuff remained in those files?

Reply:
I haven't seen any code related to py2-py3-compat except for the `__future__` imports. But your question sounds like there is something - what did I miss?

Reply:
nope, I haven't checked them ;)